### DISCUSSION TITLE
Indirizzo del server xxxxx.local non funziona

### DISCUSSION THREAD

**Discussion by angelo5619** [indomus/forum#2078] [HUB: Home Assistant]
Salve a tutti, primo post, ma utente da anni di HA.
Ho smanettato parecchio, ma a questa proprio non ci sono arrivato.
Da tempo chiamo il mio Raspberry 3, con HA core ultima versione, con il nome che gli ho assegnato in fase di installazione (2 anni) come `raspangelo.local:8123`. 
E' da circa un mese che a quell'indirizzo non risponde più. O meglio:

- se scrivo `raspangelo.local:8123` sul browser, dopo 5 minuti circa (si, 5min) mi risponde correttamente ma è lento nelle risposte.
- se scrivo `raspangelo:8123` mi risponde subito.

Se faccio un `ping `da windows:

- `ping raspangelo.local` mi risponde dopo 5 minuti e mi dà un indirizzo ipv6
- `ping raspangelo` mi risponde correttamente e mi dà un'indirizzo ipv4

Al momento entro da esterno con duckdns e tutto va a meraviglia.

Facendo `ping `da un altro pc in rete ottengo le stesse risposte ma entrambe immediate.

Non so che pesci prendere. Consigli?

grazie
Angelo

**Comment by 50494554524F** [comment]
aggiungi ip e dominio nel file "hosts"...

**Reply by angelo5619** [reply]
Beh, allora faccio prima a mettere l'IP al posto del nome. Volevo capire dov'è l'inghippo.
Comunque grazie

**Reply by 50494554524F** [reply]
dopo che avrai messo raspangelo.local nel file hosts potrai usare quello, o il dominio che ti pare...
